libminifi/src/utils/BackTrace.cpp

```diff
@@ -129,9 +129,6 @@ BackTrace TraceResolver::getBackTrace(std::string thread_name, std::thread::nati
   if (0 == thread_handle || pthread_equal(pthread_self(), thread_handle)) {
     pull_trace();
   } else {
-    if (thread_handle == 0) {
```

Comment:
It was impossible for control flow to reach this because this case was handled in the previous `if`.